shardingsphere-operator/pkg/controllers/shardingsphere_chaos_controller.go

```diff
@@ -40,7 +43,9 @@ import (
 
 const (
 	ShardingSphereChaosControllerName = "shardingsphere-chaos-controller"
-	ssChaosDefaultEnqueueTime         = 5 * time.Second
+	ssChaosDefaultEnqueueTime         = 10 * time.Second
+	defaultCreatedMessage             = " is created successfully"
```

Comment:
I may not suggest to put these messages as consts. Actually, this could be wrapped in a function matter.